truelearn/utils/visualisations/_bubble_plotter.py

```diff
@@ -47,6 +47,9 @@ def plot(
         It will not draw anything if the knowledge given by the user is empty, or
         if topics and top_n make the filtered knowledge empty.
 
+        Currently, this method requires that the mean of all knowledge components
+        within the given Knowledge be positive.
```

Comment:
Currently this method requires that the mean of all knowledge components of the users Knowledge to be positive. (rephrased)